docs/conf.py

```diff
@@ -239,22 +233,6 @@
 todo_include_todos = True  # Uncomment to show todos.
 
 
-# An extension that allows replacements for code blocks that
-# are not supported in `rst_epilog` or other substitutions.
-# https://stackoverflow.com/a/56328457/2214933
-# def source_replace(app, docname, source):
-#     result = source[0]
-#     for key in app.config.source_replacements:
-#         result = result.replace(key, app.config.source_replacements[key])
-#     source[0] = result
-
-
-# # Dict of replacements.
-# source_replacements = {
-#     "{PLONE_BACKEND_VERSION}": "6.0.4",
-# }
-
-
 def setup(app):
     app.add_config_value("source_replacements", {}, True)
     app.connect("source-read", source_replace)
```

Comment:
These three lines are also no longer necessary because they are part of `source_replace`.